README.md

```diff
@@ -52,6 +52,26 @@ make VERSION=0.3.0 push
 make clean
 ```
 
+On Windows, using the tools described below, use the command:
+
+"C:\Program Files\git\bin\bash" -c "make"
```

Comment:
I had trouble with powershell as well. Actually the easiest way to do it is to run git bash and just use make in there. But that's not as easy for appveyor to do. Agreed it would be better with PS support.